src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -459,21 +465,23 @@ def load_worse_mam_birth_prevalence(builder: Builder, cause: str) -> pd.DataFram
 
 
 # noinspection PyUnusedLocal
-def load_better_mam_exposure(builder: Builder, cause: str) -> pd.DataFrame:
-    return (
-        load_child_wasting_exposures(builder)[WASTING.CAT25]
-        .reset_index()
-        .rename(columns={WASTING.CAT25: "value"})
-    )
+def load_better_mam_exposure(builder: Builder, cause: str) -> Union[float, pd.DataFrame]:
+    exposure = load_child_wasting_exposures(builder)
+    if isinstance(exposure, pd.DataFrame):
+        exposure = (
+            exposure[WASTING.CAT25].reset_index().rename(columns={WASTING.CAT25: "value"})
+        )
+    return exposure
 
 
 # noinspection PyUnusedLocal
-def load_worse_mam_exposure(builder: Builder, cause: str) -> pd.DataFrame:
-    return (
-        load_child_wasting_exposures(builder)[WASTING.CAT2]
-        .reset_index()
-        .rename(columns={WASTING.CAT2: "value"})
-    )
+def load_worse_mam_exposure(builder: Builder, cause: str) -> Union[float, pd.DataFrame]:
```

Comment:
All of these functions are doing the same thing except on a different column - consider making a single function that takes the col as an arg